product/views/MiddlewareServerGroup.yaml

```diff
@@ -0,0 +1,82 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Middleware Server Groups
+
+# Menu name
+name: Middleware Server Groups
+
+# Main DB table report is based on
+db: MiddlewareServerGroup
+
+# Columns to fetch from the main table
+cols:
+- feed
+- name
+- profile
+
+
+# Included tables (joined, has_one, has_many) and columns
+include:
+  middleware_domain:
+    columns:
+    - name
+
+# Included tables and columns for query performance
+
+
+# Order of columns (from all tables)
+col_order:
+- name
+- feed
+- middleware_domain.name
+- profile
+
+# Column titles, in order
+headers:
+- Server Group Name
+- Feed
+- Domain Name
+- Profile
+
+# Condition(s) string for the SQL query
+conditions:
+
+# Order string for the SQL query
+order: Ascending
+
+# Columns to sort the report on, in order
+sortby:
+- name
+- profile
+- middleware_domain.name
+- feed
```

Comment:
Is there any documentation for it? The comment says "in order" but it's ambiguous... does it mean that the order needs to be the same as it was declared or that the order is important and means denotes the sorting order. I think the latter is the true, but I don't know